doc/en/developer/source/quickstart/intellij.rst

```diff
@@ -9,60 +9,75 @@ Import modules into IntelliJ
 ----------------------------
 
 #. Run the IntelliJ IDE
-#. Select ``File -> New -> Project from Existing Sources...``. 
-#. Navigate to the ``geoserver/src/pom.xml`` directory and click ``Open``.
+#. From the initial panel select ``Open``.
+
+   .. image:: img/intellij_open.png
+      :width: 500
+
+#. Navigate to the ``geoserver/src/pom.xml`` directory and click ``OK``.
 
    .. image:: img/intellij_import.png
-      :width: 600
+      :width: 350
 
-#. Click ``Next``, leaving all profiles unchecked.
+#. When asked click on ``Open as a Project``.
 
-   .. image:: img/intellij_import_profile.png
-      :width: 600
+   .. image:: img/intellij_project.png
+      :width: 350
 
-#. Click ``Next``, leaving the geoserver project checked.
+#. Optionally, depending on which platform you have, IntelliJ may ask you to ``Trust the Project``.
 
-   .. image:: img/intellij_import_project.png
-      :width: 600
+   .. image:: img/intellij_project_trust.png
+      :width: 350
 
-#. Click ``Next``, selecting the Java 11 JDK of your choice.
+#. Wait for IntelliJ to ``Sync`` the dependencies, you can follow the process from the ``Build`` tab panel on the bottom.
 
-   .. image:: img/intellij_import_jdk.png
-      :width: 600
+   .. image:: img/intellij_sync.png
+      :width: 500
 
-#. Click ``Finish``
+Finalize the GeoServer Project configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-   .. image:: img/intellij_import_finish.png
-      :width: 600
+#. Click ``File > Project Structure``.
 
-Before Running GeoServer
----------------------------
+   .. image:: img/intellij_project_structure.png
+      :width: 300
 
-GeoServer relies on other libraries that are maintained in parallel under the same project umbrella. These are GeoTools and GeoWebCache. So, for easier installation and setup of the GeoServer development environment, you may want to apply the following:
+#. Update the ``Name`` and select the correct ``SDK`` accordingly to your GeoServer version.
 
-#. Download the code of both and execute ``mvn clean install`` into them.
-    `GeoTools <https://github.com/geotools/geotools>`_
-    `GeoWebCache <https://github.com/geowebcache/geowebcache>`_
-#. Afterwards do the same in the GeoServer src folder
+   .. image:: img/intellij_project_sdk.png
+      :width: 400
+
+#. Click ``File > Settings``.
+
+   .. image:: img/intellij_project_settings.png
+      :width: 300
+
+#. From ``Build, Execution, Deployment > Compiler > Annotation Processors``, enable the ``Annotation processing``.
+
+   .. image:: img/intellij_project_settings_annotation_processing.png
+      :width: 600
```

Comment:
Here you say to enable annotations, but below, to disable them.... LOL
I checked, in my IDE they are disabled.

Reply:
@aaime moved to a dedicated Troubleshoot section